core/src/trezor/ui/layouts/tt_v2/__init__.py

```diff
@@ -981,16 +981,6 @@ async def confirm_coinjoin(
     )
 
 
-def show_coinjoin() -> None:
```

Comment:
why is this gone?

Reply:
This is handled by Busyscreen which directly calls the rust layout.

That said, the deleted code looks like:
![deleted](https://user-images.githubusercontent.com/85857/199747429-3d51e001-8e4e-492d-8190-e2758f2c274c.png)

And the busyscreen currently renders like this:
![hs](https://user-images.githubusercontent.com/85857/193703859-1c8b774f-eb7f-4b03-baa9-c4718f33bd97.png)

The former looks nicer, perhaps I should modify the homescreen code to look like it?

Reply:
`show_coinjoin()` is currently not used anywhere. The reference to it was removed in https://github.com/trezor/trezor-firmware/commit/5b3db7eca19eab2723f168563594b7eaacd4ef75, see the move from `core/src/apps/homescreen/busyscreen.py` to `core/src/trezor/ui/layouts/tt_v2/homescreen.py`. However, I also like the deleted screen better. I am wondering, should we add "Please wait " to it, or would it be too much?

Reply:
Like this?
![cj](https://user-images.githubusercontent.com/85857/212309778-a2e29f09-729d-464e-b311-670d07235e5e.png)

Reply:
Hmm, not sure, this way it slightly overshadows the main message that we want to communicate, which is "Do not disconnect your Trezor". I was thinking of the "Please wait" as a title. But I am not too familiar with the new design, so I don't want to break consistency. I leave it to @hynek-jina.

Reply:
Dunno if I'm understanding this right:
![cj2](https://user-images.githubusercontent.com/85857/212316415-03e89dcb-b7c4-4a00-9ae2-6ae5965e61dc.png)

Or did you mean to add third block of text above the icon?

Reply:
I meant it above the icon (which is what might break consistency with the new design). We can also do it in reversed order: bold "Do not disconnect your Trezor." and then under that normal font "Coinjoin in progress. Please wait." But as I said, I leave it to Hynek.

Reply:
I can do this one right now as it uses existing layout (with lowercase w):
![cj3](https://user-images.githubusercontent.com/85857/212328068-b262b418-9f58-472a-8913-a0d872928690.png)

We currently have no layout with title-icon-text1-text2 so I'd prefer if you created issue for me to create one. I think @Hannsek is now in charge of UI2 product-wise instead of Hynek?

Reply:
Coinjoin screens for UI2 are here: https://www.figma.com/file/l0gG9XeRJ8FTDQ3cfb1wv9/Trezor-T%2C-Trezor-One?node-id=5752%3A43361&t=TWzAsAeUH4v2iwf8-4

Here is the discussed screen if I'm not mistaken.
![Rounds](https://user-images.githubusercontent.com/26002916/212333290-58da95d1-64a4-427b-ab96-2c79a256b4e0.png)

**So do we need another issue? Or what is the next step, please?**

_btw. it's little bit confusing with the responsibilities.. @Hannsek is taking care about whole UI2 and I'm responsible for coinjoin... so we have to cooperate. 🙂 Final word with trezor UI is on Hans_

Reply:
I wasn't aware of this screen, yes, please open an issue.

Reply:
Screens are not yet final. So ignore it for now.. @Hannsek will open related issues once is it ready for development.